src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -235,6 +241,75 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     this.dialog.open(FilePropertiesModal, filePropConfig);
   }
 
+  showRenameField(file: any) {
+    let selectedNodes = document.getElementsByClassName('ui-treenode-content-selected');
```

Comment:
I see further down you added a conditional that checks this selectedNode against the name you're looking for, which is nearly a perfect bandage but it will break if the names of the nodes are the same

![image](https://user-images.githubusercontent.com/20528015/94846344-fad07f80-03ee-11eb-9826-d3179a75b57c.png)

This a super minor bug but I think because the solution is so simple, it's worth fixing? If $event doesn't have the HTML element, than it's probably OK to forget about it

Reply:
All issues mentioned have been resolved